lib/Xmds/Soap5.php

```diff
@@ -299,17 +299,17 @@ public function RegisterDisplay($serverKey, $hardwareKey, $displayName, $clientT
 
         // cache checks
         $cacheSchedule = $this->getPool()->getItem($this->display->getCacheKey() . '/schedule');
-        $displayElement->setAttribute('checkSchedule', $cacheSchedule->isHit() == true ? "false" : "true");
+        $displayElement->setAttribute('checkSchedule', ($cacheSchedule->isHit() ? hash('crc32', $cacheSchedule->get()) : ""));
 
         $cacheRF = $this->getPool()->getItem($this->display->getCacheKey() . '/requiredFiles');
-        $displayElement->setAttribute('checkRf', $cacheRF->isHit() == true ? "false" : "true");
+        $displayElement->setAttribute('checkRf', ($cacheSchedule->isHit() ? hash('crc32', $cacheRF->get()) : ""));
```

Comment:
shouldn't that be `$cacheRF->isHit() ?`  🤔

Reply:
yes